django/urls/base.py

```diff
@@ -88,7 +89,14 @@ def reverse(viewname, urlconf=None, args=None, kwargs=None, current_app=None):
         if ns_pattern:
             resolver = get_ns_resolver(ns_pattern, resolver)
 
-    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
+    url = force_text(iri_to_uri(resolver._reverse_with_prefix(
```

Comment:
Why is the result of `iri_to_uri` stored in a variable named `url` instead of `uri`?